src/core/forces.cpp

```diff
@@ -107,8 +109,9 @@ void force_calc() {
   // to reassemble the cell lists after all position updates, also of virtual
   // particles.
   if ((lattice_switch & LATTICE_LB) &&
-      cell_structure.type == CELL_STRUCTURE_DOMDEC && (!cell_structure.use_verlet_list))
-    cells_update_ghosts();
+      cell_structure.type == CELL_STRUCTURE_DOMDEC && (!cell_structure.use_verlet_list) &&
```

Comment:
Can you explain to me why this is needed? This seems strange to me as verlet lists have no influence on either VS or LB.